src/static/js/animal.js

```diff
@@ -873,6 +885,7 @@ $(function() {
             if (config.bool("DontShowSize")) { $("#sizerow").hide(); }
             if (config.bool("DontShowWeight")) { $("#kilosrow, #poundsrow").hide(); }
             if (config.bool("DontShowMicrochip")) { $("#microchiprow, #microchiprow2").hide(); }
+            if (config.bool("DontShowMicrochipStatus")) { $("#microchipstatus, #microchipstatus2").hide(); }
```

Comment:
DontShowMicrochipStatus should also be set to "Yes" by default in the dict at the top of configuration.py